src/core/dom/query.js

```diff
@@ -414,3 +414,26 @@ export function querySelectorInSlot(slot, selector) {
   }
   return null;
 }
+
+/**
+ * Finds a matching node inside an HTML template slot's children
+ * @param {HTMLSlotElement} slot
+ * @param {string} selector
+ * @return {HTMLElement[]}
+ */
+export function querySelectorAllInSlot(slot, selector) {
+  const nodes = /** @type {HTMLElement[] } */ (slot.assignedElements());
+
+  const list = [];
+  for (let i = 0; i < nodes.length; i++) {
+    const node = nodes[i];
+    if (matches(node, selector)) {
+      list.push(node);
+    }
+    const child = scopedQuerySelector(node, selector);
```

Comment:
If this is `querySelectorAllInSlot`, shouldn't this be `scopedQuerySelectorAll`?

Reply:
Because we are already iterating through each node, it didn't make a difference in this use case, but I will change this just in case it makes a difference in the future.